react-native-gutenberg-bridge/android/src/main/java/org/wordpress/mobile/WPAndroidGlue/WPAndroidGlueCode.java

```diff
@@ -305,6 +309,13 @@ public void performRequest(String pathFromJS, Consumer<String> onSuccess, Consum
             public void requestImageFullscreenPreview(String mediaUrl) {
                 mOnImageFullscreenPreviewListener.onImageFullscreenPreviewClicked(mediaUrl);
             }
+
+            @Override
+            public void getPreferredColorScheme(
+                    PreferredColorSchemeReceivedCallback preferredColorSchemeReceivedCallback) {
+                String preferredColorScheme = mOnPreferredColorSchemeListener.getPreferredColorScheme();
+                preferredColorSchemeReceivedCallback.onPreferredColorSchemeReceived(preferredColorScheme);
```

Comment:
I'm curious if we could avoid using a callback here and instead just return the color scheme `String` value directly? I feel like that helps simplify the code. [Something like this](https://github.com/wordpress-mobile/gutenberg-mobile/commit/089181305e88cd4bb782c5d4249e654406a426f0) is what I had in mind.

Reply:
Hey @mchowning 👋 
It seems that I changed a lot of code here so if mind doing another iteration of review?
Thanks a lot!